src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java

```diff
@@ -32,7 +32,35 @@ public Measure overwriteExistingMeasure(
       transferredMeasure.setId(mostRecentMeasure.getId());
       if (GroupPopulationUtil.areGroupsAndPopulationsMatching(
           mostRecentMeasure.getGroups(), transferredMeasure.getGroups())) {
-        transferredMeasure.setTestCases(mostRecentMeasure.getTestCases());
+        List<TestCase> originalTestCases = mostRecentMeasure.getTestCases();
+        if (CollectionUtils.isNotEmpty(originalTestCases)) {
+          // test cases that need to be carried over to the new transferred measure,
+          // however, groupPopulations need to be wiped out because there is no way
+          // to map the group ids to multiple groups, if the group ids are different
+          // then execution will blow up. Users have to manually input expected values
+          originalTestCases =
+              originalTestCases.stream()
+                  .map(
+                      testCase ->
+                          TestCase.builder()
+                              .id(testCase.getId())
+                              .name(testCase.getName())
+                              .title(testCase.getTitle())
+                              .series(testCase.getSeries())
+                              .description(testCase.getDescription())
+                              .createdAt(testCase.getCreatedAt())
+                              .createdBy(testCase.getCreatedBy())
+                              .lastModifiedAt(testCase.getLastModifiedAt())
+                              .lastModifiedBy(testCase.getLastModifiedBy())
+                              .resourceUri(testCase.getResourceUri())
+                              .validResource(testCase.isValidResource())
+                              .json(testCase.getJson())
+                              .patientId(testCase.getPatientId())
+                              .hapiOperationOutcome(testCase.getHapiOperationOutcome())
+                              .build())
```

Comment:
Lombok provides a shortcut for duplicating objects:
```suggestion
                    testCase -> testCase.toBuilder().build())
```

Reply:
@jkotanchik-SB thanks so much will do.

Reply:
@jkotanchik-SB I did the change can I resolve this conversation? Thanks